apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -1,67 +1,74 @@
 // Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT-0
 
-import React, { useState, useContext, ChangeEvent } from 'react';
-import { useHistory } from 'react-router-dom';
+import React, {ChangeEvent, useContext, useState} from 'react';
+import {useHistory} from 'react-router-dom';
 import {
-  Input,
   Checkbox,
+  DeviceLabels,
   Flex,
-  Heading,
   FormField,
-  PrimaryButton,
-  useMeetingManager,
+  Heading,
+  Input,
   Modal,
   ModalBody,
   ModalHeader,
-  DeviceLabels,
-  Select
+  PrimaryButton,
+  Select,
+  useMeetingManager
 } from 'amazon-chime-sdk-component-library-react';
 
-import { getErrorContext } from '../../providers/ErrorProvider';
+import { DefaultBrowserBehavior } from 'amazon-chime-sdk-js';
+
+import {getErrorContext} from '../../providers/ErrorProvider';
 import routes from '../../constants/routes';
 import Card from '../../components/Card';
 import Spinner from '../../components/icons/Spinner';
 import DevicePermissionPrompt from '../DevicePermissionPrompt';
 import RegionSelection from './RegionSelection';
-import { fetchMeeting, createGetAttendeeCallback } from '../../utils/api';
-import { useAppState } from '../../providers/AppStateProvider';
-import { MeetingMode, BlurValues } from '../../types';
+import {createGetAttendeeCallback, fetchMeeting} from '../../utils/api';
+import {useAppState} from '../../providers/AppStateProvider';
+import {BlurValues, MeetingMode} from '../../types';
+import meetingConfig from '../../meetingConfig';
 
 const BLUR_OPTIONS = [
-  { value: BlurValues.blurDisabled, label: "Disable Blur" }, 
-  { value: BlurValues.blur10Percent, label: "Blur CPU 10%" }, 
-  { value: BlurValues.blur20Percent, label: "Blur CPU 20%" }, 
+  { value: BlurValues.blurDisabled, label: "Disable Blur" },
+  { value: BlurValues.blur10Percent, label: "Blur CPU 10%" },
+  { value: BlurValues.blur20Percent, label: "Blur CPU 20%" },
   { value: BlurValues.blur40Percent, label: "Blur CPU 40%" },
 ];
 
 const MeetingForm: React.FC = () => {
   const meetingManager = useMeetingManager();
   const {
-    setAppMeetingInfo,
-    region: appRegion,
-    meetingId: appMeetingId,
+    region,
+    meetingId,
+    localUserName,
+    meetingMode,
+    enableSimulcast,
+    priorityBasedPolicy,
     isWebAudioEnabled,
     blurOption: blurValue,
     setBlurValue,
     toggleWebAudio,
-    setMeetingMode
+    toggleSimulcast,
+    togglePriorityBasedPolicy,
+    setMeetingMode,
+    setMeetingId,
+    setLocalName,
```

Comment:
nit: setLocalName -> setLocalUserName from React state naming convention.

Reply:
Whats naming convention? The setLocalName is an existing function so I did not change it.